js/flightlog_fields_presenter.js

```diff
@@ -411,10 +411,10 @@ function FlightLogFieldPresenter() {
             },
             'FF_INTERPOLATED' : {
                             'debug[all]':'FF Interpolated', 
-                            'debug[0]':'Setpoint Delta Impl [Roll]',
-                            'debug[1]':'Boost Amount',
-                            'debug[2]':'Boost Amount Clip [Roll]',
-                            'debug[3]':'Clip',
+                            'debug[0]':'Raw FF Derivative [Roll]',
+                            'debug[1]':'Cleaned FF Derivative ',
+                            'debug[2]':'Cleaned Boost Amount [Roll]',
+                            'debug[3]':'Duplicate Marker',
```

Comment:
Here we have the habitual nightmare of the blackbox and configurator... we need to maintain backward compatibility :(

I don't remember to have done this before for the debug names, so I can't point to a sample at this moment... but if the "old" name is correct for old logs, then we can't simply rename them.

Reply:
I see.  

The current names aren't quite right in a lot of debug cases.  In this case the existing names aren't right for 4.2 either :-)

If its tough to provide a compatibility solution for the debugs, could we make 'releases' that are correct for a given version of the software, in terms of debug names and waveform scaling ?

Reply:
I doubt. A lot of people have different quads some of them in different versions, and expect all to be working... @mikeller what do you think in this case?
I cant try to look at the code this week to see if we can make this conditional depending on the version.